legacy/oled.c

```diff
@@ -181,23 +180,23 @@ void oledInit() {
 void oledClear() { memzero(_oledbuffer, sizeof(_oledbuffer)); }
 
 void oledInvertDebugLink() {
```

Comment:
In the current change, the function body is still defined in the production firmware build. However, the function isn't called, so the compiler likely ignores it for the bootloader/firmware build and this difference isn't important.

Reply:
it looks slightly cleaner to define out the whole function, but it's called twice so that would be ifdef'ing out two call sites. keeping the function empty and letting the emulator do the job seems better overall